assets/js/app.js

```diff
@@ -1,8 +1,10 @@
 //declare variables
-var city1;
-var city2;
-var country1;
-var country2;
+var home;
```

Comment:
//declare variables
var city1;
var city2;
var country1;
var country2;
var APIKeyWeather = "30bacb09b1a6ec05e34d8f5ac74ca7de";
var queryURL;
var sumTemp = 0;
var sumClouds = 0;
var sumRain = 0;
var aveTemp;
var aveClouds;
var aveRain;

this should be kept as the weather variables at the global level. these need to be added back in #49